CRM/Case/BAO/Case.php

```diff
@@ -1212,9 +1212,20 @@ public static function getCaseActivity($caseID, &$params, $contactID, $context =
    *
    */
   public static function getRelatedContacts($caseID, $skipDetails = FALSE) {
```

Comment:
I think I wanna get all @xurizaemon on you & complain about the double negative. Can we do $includeDetails for readability